cypress/integration/tests/controls/stakeholders/crud.test.ts

```diff
@@ -140,16 +148,16 @@ describe("Stakeholder CRUD operations", () => {
 
         // Delete stakeholder
         stakeholder.delete();
+        cy.wait("@deleteStakeholder");
         cy.wait("@getStakeholders");
 
```

Comment:
do we need to go with both interceptors? is there any error you noticed with only `@deleteStakeholder`

Reply:
Delete will wait for deletion, and after delete, the table is reloaded in DOM using GET api call, so it gives surety of table being loaded after delete